### PR TITLE
articles routes

### PR DESCRIPTION
**Pull request by BehnazDer**

### CODE REVIEW

File: src/routes/articlesRoutes.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const Article = require('../models/Article');
+const Articles = require('../models/Article');
```
Comment (thdvbr) [inline src/routes/articlesRoutes.js]: models/Article already required

File: src/models/Article.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      required: true,
+    },
+
+    urlToImage: {
+      type: String,
+      required: true,
+    },
+    violence_type: {
+      type: String,
+      required: true,
```
Comment (irina-baeva) [inline src/models/Article.js:24]: do u use the type violence in FE? it has to be array in the future

File: src/routes/articlesRoutes.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        return res.send(foundArticles);
+      }
+      return res.send(err);
+    });
+  });
+
+// Getting a specific article
+router.get('/articles/:id', function (req, res) {
+    console.log('get req for single article');
+    Article.findById(req.params.id)
```
Comment (irina-baeva) [inline src/routes/articlesRoutes.js]: console logs

File: src/routes/articlesRoutes.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    .exec(function(err, article){
+        if (err){
+            console.log("error");
```
Comment (irina-baeva) [inline src/routes/articlesRoutes.js]: return res.send(err) or console.error(err)

File: src/models/Article.js (hunk 2 above)
Comment (BehnazDer) [reply src/models/Article.js:24]: I changed it to array

File: src/routes/articlesRoutes.js (hunk 4 above)
Comment (BehnazDer) [reply src/routes/articlesRoutes.js]: fixed

File: src/routes/articlesRoutes.js (hunk 1 above)
Comment (BehnazDer) [reply src/routes/articlesRoutes.js]: Both should be there, otherwise the code breaks

**Pull request merged**